mobile/src/components/course/components/course-header/course-header.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from 'react';
+
+import defaultCourseImage from '~/assets/images/default-course-image.png';
+import { CourseGetResponseDto } from '~/common/types/types';
+import {
+  Category,
+  Icon,
+  Image,
+  Pressable,
+  Text,
+  View,
+} from '~/components/common/common';
+import { getImageUri } from '~/helpers/helpers';
+import { useCallback, useFocusEffect, useState } from '~/hooks/hooks';
+
+import { CourseDescription } from '../components';
+import { styles } from './styles';
+
+type Props = {
+  width: number;
+  course: CourseGetResponseDto;
+  hasEditCategoryPermission: boolean;
+  onEditModeToggle: () => void;
+};
+
+const CourseHeader: FC<Props> = ({
```

Comment:
Let's rename it to something like CourseContent (don't forget about filename, folder name)